Dockerfile

```diff
@@ -27,8 +27,8 @@ ENV NODE_ENV production
 ENV WORKERS $workers
 ENV BUILD_TRANSLATION_CHUNKS true
 ENV CHROMEDRIVER_SKIP_DOWNLOAD true
-ENV PUPPETEER_SKIP_DOWNLOAD true
-ENV PLAYWRIGHT_SKIP_DOWNLOAD true
+ENV PUPPETEER_SKIP_DOWNLOAD false
+ENV PLAYWRIGHT_SKIP_DOWNLOAD false
```

Comment:
Looks like this file has changed since you created this PR. You may need to _rebase_ the branch.